pagopa/io-app
# feat: [IOPID-971,IOPID-1074]  Add IntesiGroup IdP local reference

## Short description
This PR adds:
- the `RelayState` query parameter to SPID IdP login URI: https://api-app.io.pagopa.it/login?authLevel=SpidL2&entityID=intesiid&RelayState=appio
- the Intesi Group SPID IdP to local IdP list

And it updates the `list.json` to `list2.json` that is the new [io-services-metadata](https://github.com/pagopa/io-services-metadata/pull/739) IdP remote list, so that only the IO APP versions handling the `RelayState` query param will show the Intesi Group SPID IdP: this IdP doesn't work without the aforementioned query parameter. 

## How to test
Run the application with `.env.production` and check that in the SPID IdP list is present the Intesi Group SPID IdP, among the others.

<details><summary>IdP list with Intesi Group</summary>
<p>

| IdP list with Intesi Group |
| - |
| <img src="https://github.com/pagopa/io-app/assets/16268789/ad66f124-95a9-4ca1-81ee-3dcf20d5d656" width="250" /> |



</p>
</details>
